src/Panel/File.php

```diff
@@ -126,69 +126,9 @@ public function dragText(
 	/**
 	 * Provides options for the file dropdown
 	 */
-	public function dropdown(array $options = []): array
+	public function dropdown(): array
```

Comment:
`Comment:` I did not migrate the feature of passing a `$options` argument to the new controller classes. I could not find any use case for passing extra options.

Reply:
Totally possible that we never used it at all.